contracts/multisig-prover/src/contract.rs

```diff
@@ -41,6 +41,7 @@ pub fn instantiate(
         service_name: msg.service_name,
         chain_name: ChainName::from_str(&msg.chain_name)
             .map_err(|_| ContractError::InvalidChainName)?,
+        worker_set_diff_threshold: msg.worker_set_diff_threshold,
```

Comment:
What is this?

Reply:
It's the max number of workers that can change without triggering a worker set update.